packages/flutter/lib/src/material/i18n/localizations.dart

```diff
@@ -311,12 +311,14 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
     "previousMonthTooltip": r"Vorheriger Monat",
     "nextPageTooltip": r"Nächste Seite",
     "previousPageTooltip": r"Vorherige Seite",
-    "showMenuTooltip": r"Zeige das Menü",
+    "showMenuTooltip": r"Menü anzeigen",
     "licensesPageTitle": r"Lizenzen",
     "pageRowsInfoTitle": r"$firstRow–$lastRow von $rowCount",
     "pageRowsInfoTitleApproximate": r"$firstRow–$lastRow von etwa $rowCount",
     "rowsPerPageTitle": r"Zeilen pro Seite:",
-    "selectedRowCountTitleOther": r"$selectedRowCount Artikel ausgewählt",
+    "selectedRowCountTitleZero": r"Keine Objekte ausgewält",
```

Comment:
Here and the two below: There is an "h" missing. It's "ausgewählt". Sorry, I think I misspelled that in my previous comment... :(